### PR TITLE
fix: Added AWS, GCP and Azure Data Sources to recommended tab

### PR DESCRIPTION
**Pull request by omkarK06**
#5038 

## Release Notes

- **New Features**
  - Introduced new tabs for AWS, GCP, and Azure configurations in the ingestion section.
  - Added a new component to display AWS configuration details with service documentation links.
  - Enhanced GCP configuration component with additional documentation links.
  - Updated Azure configuration component layout and links.

- **Bug Fixes**
  - Removed outdated ingestion options for Kinesis Firehose and GCP Logs from the interface.

- **Refactor**
  - Simplified route structure by removing obsolete components and adding new ones for AWS, GCP, and Azure.

### CODE REVIEW

File: web/src/components/ingestion/recommended/GcpConfig.vue [hunk 1]
```diff
@@ -17,6 +17,20 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
+      <div class="tw-py-2">
+        Pub/Sub Logs -
+        <a
+          href="https://openobserve.ai/blog/send-gcp-logs-to-openobserve"
+          class="hover:tw-underline text-primary"
+        >
+          https://openobserve.ai/blog/send-gcp-logs-to-openobserve
+        </a>
+      </div>
+    </div>
```
[the remark this answers is not in the document]
Comment (omkarK06) [reply web/src/components/ingestion/recommended/GcpConfig.vue]: Done

File: web/src/components/ingestion/recommended/AzureConfig.vue [hunk 2]
```diff
@@ -28,7 +40,7 @@ import { getImageURL } from "../../../utils/zincutils";
 import CopyContent from "@/components/CopyContent.vue";
 
 export default defineComponent({
-  name: "kineses-firehose",
+  name: "AzureConfig",
```
[the remark this answers is not in the document]
Comment (omkarK06) [reply web/src/components/ingestion/recommended/AzureConfig.vue:41]: Done

File: web/src/components/ingestion/recommended/AWSConfig.vue [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    endpoint.value = {
+      url: store.state.API_ENDPOINT,
+      host: url.hostname,
+      port: url.port || (url.protocol === "https:" ? "443" : "80"),
+      protocol: url.protocol.replace(":", ""),
+      tls: url.protocol === "https:" ? "On" : "Off",
+    };
+
+    const content = `HTTP Endpoint: ${endpoint.value.url}/aws/${store.state.selectedOrganization.identifier}/default/_kinesis_firehose
+Access Key: [BASIC_PASSCODE]`;
```
[the remark this answers is not in the document]
Comment (omkarK06) [reply web/src/components/ingestion/recommended/AWSConfig.vue]: There will be always one selected org

File: web/src/components/ingestion/recommended/AWSConfig.vue [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+      tls: "",
+    });
+    const url = new URL(store.state.API_ENDPOINT);
+    endpoint.value = {
+      url: store.state.API_ENDPOINT,
+      host: url.hostname,
+      port: url.port || (url.protocol === "https:" ? "443" : "80"),
+      protocol: url.protocol.replace(":", ""),
+      tls: url.protocol === "https:" ? "On" : "Off",
+    };
```
[the remark this answers is not in the document]
Comment (omkarK06) [reply web/src/components/ingestion/recommended/AWSConfig.vue]: Added try catch and todo for interface

File: web/src/components/ingestion/recommended/AWSConfig.vue [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+
+export default defineComponent({
+  name: "AWSConfig",
+  props: {
+    currOrgIdentifier: {
+      type: String,
+    },
+    currUserEmail: {
+      type: String,
+    },
```
[the remark this answers is not in the document]
Comment (omkarK06) [reply web/src/components/ingestion/recommended/AWSConfig.vue:57]: Noted

**Review by bjp232004 — APPROVED** [review]

**Pull request merged**